LAR Integration Part II

PordyHUN opened the pull request:
I Fixed the errors i caused in the decisions, and progressed further.

PordyHUN committed:
Bertalan Árpád, Historical commander of the hungarian paratroopers.

WW I war hero, similar path and achevement like Rommel did in the WW I.
Died in an take-off accident when Hungary enters war against yugoslavia.

PordyHUN committed:
Fix on maritime stats and a few new DDS

PordyHUN committed:
Merge branch 'Beta-Edits' into Pordy's_Toolbox

featherus committed:
Fixed Map Issues

featherus committed:
Many 1.9 File Changes

featherus committed:
Every nation a focus filter

PordyHUN committed:
1.9 integration progress I

PordyHUN committed:
Merge remote-tracking branch 'origin/LaResistanceUpgrade' into Pordy's_Toolbox

PordyHUN committed:
Merge branch 'Beta-Edits' into Pordy's_Toolbox

PordyHUN committed:
LAR compatibility Part I

PordyHUN committed:
Merge branch 'Beta-Edits' into Pordy's_Toolbox

PordyHUN committed:
Merge branch 'Beta-Edits' into Pordy's_Toolbox

PordyHUN committed:
LAR Integration Part II

PordyHUN committed:
Update resource_prospecting.txt

PordyHUN committed:
Update HOL.txt

Fixed wrong focus scoop.

PordyHUN committed:
Clearing the error log

Clearing the error log

PordyHUN committed:
LAR Integration Additions

PordyHUN committed:
Clearing the error log Again

PordyHUN committed:
further progress on integration an bughunt

Thinking-waffle commented:
History modifications are outdated (when related to leaders not techs or ship variants), reproduce paradox mistakes or some dubious choices that we don't want to make.

PordyHUN committed:
reverting some silly changes

PordyHUN committed:
Reverting another two history files to the original (not vanilla) state

PordyHUN committed:
History units update in 1.9.1

PordyHUN committed:
1.9.1 stuff

PordyHUN committed:
Merge branch 'Beta-Edits' into Pordy's_Toolbox

PordyHUN committed:
Motorized recon support for motorized divisions

PordyHUN committed:
* Bertalan Árpád trough decision

* Bertalan Árpád trough decision
* Fixed non-motorized units mot_recon usage.

PordyHUN committed:
MTG - NON MTG naval techs for every country

PordyHUN committed:
Delete SPR.txt

Identical to vanilla

PordyHUN committed:
Ideas

Our idea files now follow the vanilla changes of 1.9.1.
removed vanilla-like files.
Dont belive of the compare of the gamerules file. Just changed the order not the content, to easier compare to vanilla.

PordyHUN committed:
All focus generated wargoals last forever. Tweak your individual wargoals if you dont agree.

PordyHUN committed:
finishing the common folder

PordyHUN committed:
Removed all vanilla identical division name file.

PordyHUN committed:
Events

PordyHUN committed:
Revert "Delete SPR.txt"

This reverts commit f50f91d25f8a224760f6a197aba2971e709c0444.

PordyHUN committed:
Clening the error log
